assets/CET_to_py.py

```diff
@@ -6,9 +6,11 @@
 see https://github.com/pyviz/colorcet/issues/11 for more details
 """
 
-import os, os.path, csv
+import csv
+import re
+from pathlib import Path
 
-paths = ['CETperceptual_csv_0_1', 'CETperceptual_csv_0_1_v2', 'Glasbey']
+csv_folders = ['CETperceptual_csv_0_1', 'Glasbey']
```

Comment:
> > I added a # cmap_def comment to each colormap definition in **init**.py to easily find them with a regex.
> 
> That's ok if it really helps some other tooling, but it doesn't seem necessary to me, since `^[a-zA-Z].* = [[]` already matches the same set of lines.

I guess I was concerned about the potential fragility of the regex you mentioned (i.e. capturing something besides a colormap definition), but yeah it would probably have been fine to just do that.